src/views/CGI/CGI.tsx

```diff
@@ -143,15 +143,15 @@ const CgiProductPage = (props: { title: string }) => {
             </p>
 
             <h2>Data Handling</h2>
-            <p>
+            <div>
               The following rounding of data are used for the Index calculation
               and publication:
               <ul>
                 <li>Prices are rounded to eight significant figures</li>
                 <li>Weights are rounded to four decimal places</li>
                 <li>Index levels are rounded to two decimal places</li>
               </ul>
-            </p>
+            </div>
```

Comment:
does this change how the page is rendered?

Reply:
i.e. is the spacing still good/does it need to be fixed with further styling

Reply:
it's good yeah